packages/jitm/src/JITM.php

```diff
@@ -419,34 +419,46 @@ function dismiss( $id, $feature_class ) {
 	/**
 	 * Asks the wpcom API for the current message to display keyed on query string and message path
 	 *
-	 * @param $message_path string The message path to ask for
-	 * @param $query string The query string originally from the front end
+	 * @param string $message_path The message path to ask for.
+	 * @param string $query The query string originally from the front end.
 	 *
 	 * @return array The JITM's to show, or an empty array if there is nothing to show
 	 */
-	function get_messages( $message_path, $query ) {
-		// custom filters go here
-		add_filter( 'jitm_woocommerce_services_msg', array( 'Jetpack_JITM', 'jitm_woocommerce_services_msg' ) );
-		add_filter( 'jitm_jetpack_woo_services_install', array( 'Jetpack_JITM', 'jitm_jetpack_woo_services_install' ) );
+	public function get_messages( $message_path, $query ) {
+		// Custom filters go here.
+		add_filter(
+			'jitm_woocommerce_services_msg',
+			array(
+				'Automattic\Jetpack\JITM',
+				'jitm_woocommerce_services_msg',
+			)
+		);
+		add_filter(
+			'jitm_jetpack_woo_services_install',
+			array(
+				'Automattic\Jetpack\JITM',
+				'jitm_jetpack_woo_services_install',
+			)
+		);
 		add_filter(
 			'jitm_jetpack_woo_services_activate',
 			array(
-				'Jetpack_JITM',
+				'Automattic\Jetpack\JITM',
 				'jitm_jetpack_woo_services_activate',
 			)
 		);
```

Comment:
Since we are in that class already, couldn't we do 

```php
add_filter( 'jitm_woocommerce_services_msg', array( $this, 'jitm_woocommerce_services_msg' ) );
add_filter( 'jitm_jetpack_woo_services_install', array( $this, 'jitm_jetpack_woo_services_install' ) );
add_filter( 'jitm_jetpack_woo_services_activate', array( $this, 'jitm_jetpack_woo_services_activate' ) );
```

Reply:
Yes - thanks! See commit d101760